llvm/lib/Target/AMDGPU/SILateBranchLowering.cpp

```diff
@@ -116,14 +117,62 @@ static void splitBlock(MachineBasicBlock &MBB, MachineInstr &MI,
   MDT->applyUpdates(DTUpdates);
 }
 
-void SILateBranchLowering::expandChainCall(MachineInstr &MI) {
+static void copyOpWithoutRegFlags(MachineInstrBuilder &MIB,
+                                  MachineOperand &Op) {
+  if (Op.isReg())
+    MIB.addReg(Op.getReg());
+  else
+    MIB.add(Op);
```

Comment:
There's probably a cleaner way to do this, but we should really finish the removal of kill flags